### DISCUSSION TITLE
Switch to enable/disable differential thrus

### DISCUSSION THREAD

**Discussion by samuraitheta** [iNavFlight/inav#7024] [Q&A]
I just installed a Matek F405 on my twin engine Flitetest Explorer, and I am hoping to figure out a way to enable and disable differential thrust on a switch. I see how to setup logical conditions with servos in the configurator, but I can't figure out how to apply conditions to the motor section of the mixer. Is this possible?

**Comment by DzikuVx** [comment]
There is no way to disable/enable differentlial thrust with a switch. It's either configured with a mixer, or not

**Comment by samuraitheta** [comment]
I love your videos! Thanks for the reply.

Do you know if I could submit this as a feature request? I used to setup a switch in open TX to turn my differential thrust on and off. 

Matthew Andrew
(775)391-5616
